packages/@react-aria/select/src/useSelect.ts

```diff
@@ -85,6 +91,8 @@ export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>,
   let triggerProps = mergeProps(mergeProps(menuTriggerProps, fieldProps), typeSelectProps);
   let valueId = useId();
 
+  let onKeyDown = props.onKeyDown ? chain(triggerProps.onKeyDown, props.onKeyDown) : triggerProps.onKeyDown;
```

Comment:
is that the right way todo it or is there a better way?

Reply:
chaining with undefined is fine, it'll just skip those https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/utils/src/chain.ts#L19